.lando/zzz-lando-my-custom.ini

```
@@ -10,24 +10,18 @@ memory_limit = 4G
 ; Xdebug
 xdebug.max_nesting_level = 256
 xdebug.show_exception_trace = 0
-xdebug.collect_params = 0
 ; Extra custom Xdebug setting for debug to work in VSCode.
-xdebug.remote_enable = 1
-xdebug.remote_autostart = 1
-xdebug.remote_host = ${LANDO_HOST_IP}
-xdebug.remote_connect_back = 1
-xdebug.remote_log = /tmp/xdebug.log
+xdebug.client_host = ${LANDO_HOST_IP}
+xdebug.discover_client_host = 1
 
 ; xdebug3 config.  See https://xdebug.org/docs/upgrade_guide
-xdebug.collect_assignments=1
-xdebug.collect_includes=1
-xdebug.collect_vars=1
-xdebug.force_display_errors=1
-xdebug.force_error_reporting=1
-
-xdebug.mode=debug,trace,develop
-xdebug.start_with_request=yes
-xdebug.client_port=9001
+xdebug.collect_assignments = 1
+xdebug.force_display_errors = 1
+xdebug.force_error_reporting = 1
+xdebug.mode = debug,trace,develop,
+xdebug.start_with_request = yes
+;xdebug.client_port = 9000
```

```suggestion
xdebug.client_port = 9003
```
for sake of explicity?

---

It does not work on my local if I rebuild with this value explicitly set.  I could only get it to work on 9000 when explicitly set.

---

Just tried it again to make sure, and it worked.  So I updated it.